src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -514,6 +514,7 @@ void DiscoveryImplPlatform::HandleNodeResolve(void * context, DnssdService * res
     DiscoveryImplPlatform * mgr = static_cast<DiscoveryImplPlatform *>(context);
     DiscoveredNodeData data;
     Platform::CopyString(data.hostName, result->mHostName);
+    Platform::CopyString(data.instanceName, result->mName);
```

Comment:
Is this being used anywhere? Might just be my bad view of diffs...

Reply:
Used in `DiscoveredNodeData::LogDetail` and in the chip-tool and Python bits.